config/settings.yml

```diff
@@ -11,6 +11,9 @@ site_name: Helpy Support # This is the name of your support site
 site_tagline: Support and Helpdesk Software # This is the tagline of your support site
 product_name: # This is the product name your are supporting
 support_phone: # This is the support number to be displayed
+app_favico: favicon.ico #Full or partial url for app favicon
```

Comment:
@popperz0r I was getting ready to merge this in, when I noticed the Setting variable is `app_favico` instead of favicon.  Is this Portuguese or a typo?  Can you name the variable `app_favicon` for consistency with the rest of the file?  I see it is even spelled favicon in the comment you wrote :)  If you change that and resubmit we'll be good to go!